src/web/releases.rs

```diff
@@ -439,13 +386,12 @@ pub fn author_handler(req: &mut Request) -> IronResult<Response> {
                 cexpect!(req, author.nth(1)),
             )
         } else {
-            get_releases_by_author(&mut conn, page_number, RELEASES_IN_RELEASES, author)
+            return Err(Nope::AuthorNameInvalid.into());
```

Comment:
Do you think it would be better to show the owner even without an `@` sign? That seems more helpful.

If not, the error should be more accurate and say that only owners are shown, not authors. "Invalid author" isn't very helpful for finding what went wrong.

Reply:
> Do you think it would be better to show the owner even without an `@` sign? That seems more helpful.

Make sense.